src/lamp_py/tableau/jobs/glides.py

```diff
@@ -0,0 +1,249 @@
+        ("id", pyarrow.large_string()),
+        ("type", pyarrow.large_string()),
+        ("time", pyarrow.date32()),  # yyyy-mm-ddT hh:mm:ssZ timestamp (UTC) -> Datetime (EST)
+        ("source", pyarrow.large_string()),
+        ("specversion", pyarrow.large_string()),
+        ("dataschema", pyarrow.large_string()),
+        ("data.tripUpdates.previousTripKey.serviceDate", pyarrow.date32()),  # YYYY-MM-DD str -> date
+        ("data.tripUpdates.previousTripKey.tripId", pyarrow.large_string()),
+        ("data.tripUpdates.previousTripKey.startLocation.gtfsId", pyarrow.large_string()),
+        ("data.tripUpdates.previousTripKey.startLocation.todsId", pyarrow.large_string()),
+        ("data.tripUpdates.previousTripKey.endLocation.gtfsId", pyarrow.large_string()),
+        ("data.tripUpdates.previousTripKey.endLocation.todsId", pyarrow.large_string()),
+        ("data.tripUpdates.previousTripKey.startTime", pyarrow.time32("s")),  # HH:MM:SS str -> time
+        ("data.tripUpdates.previousTripKey.endTime", pyarrow.time32("s")),  # HH:MM:SS str -> time
+        ("data.tripUpdates.previousTripKey.revenue", pyarrow.large_string()),
+        ("data.tripUpdates.previousTripKey.glidesId", pyarrow.large_string()),
+        ("data.tripUpdates.type", pyarrow.large_string()),
+        ("data.tripUpdates.tripKey.serviceDate", pyarrow.date32()),  # YYYY-MM-DD str -> date
+        ("data.tripUpdates.tripKey.tripId", pyarrow.large_string()),
+        ("data.tripUpdates.tripKey.startLocation.gtfsId", pyarrow.large_string()),
+        ("data.tripUpdates.tripKey.startLocation.todsId", pyarrow.large_string()),
+        ("data.tripUpdates.tripKey.endLocation.gtfsId", pyarrow.large_string()),
+        ("data.tripUpdates.tripKey.endLocation.todsId", pyarrow.large_string()),
+        ("data.tripUpdates.tripKey.startTime", pyarrow.time32("s")),  # HH:MM:SS str -> time
+        ("data.tripUpdates.tripKey.endTime", pyarrow.time32("s")),  # HH:MM:SS str -> time
+        ("data.tripUpdates.tripKey.revenue", pyarrow.large_string()),
+        ("data.tripUpdates.tripKey.glidesId", pyarrow.large_string()),
+        ("data.tripUpdates.comment", pyarrow.large_string()),
+        ("data.tripUpdates.startLocation.gtfsId", pyarrow.large_string()),
+        ("data.tripUpdates.startLocation.todsId", pyarrow.large_string()),
+        ("data.tripUpdates.endLocation.gtfsId", pyarrow.large_string()),
+        ("data.tripUpdates.endLocation.todsId", pyarrow.large_string()),
+        ("data.tripUpdates.startTime", pyarrow.time32("s")),  # HH:MM:SS str -> time
+        ("data.tripUpdates.endTime", pyarrow.time32("s")),  # HH:MM:SS str -> time
+        ("data.tripUpdates.cars", pyarrow.large_string()),
+        ("data.tripUpdates.revenue", pyarrow.large_string()),
+        ("data.tripUpdates.dropped", pyarrow.large_string()),
+        ("data.tripUpdates.scheduled", pyarrow.large_string()),
+    ]
+)
+
+glides_operator_signed_in_schema = pyarrow.schema(
```

Comment:
This schema doesn't match what you're outputting from `create_operator_signed_in_glides_parquet`